### PR TITLE
Associated with #14595

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14803/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14803/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/Meshes/Node/Blocks/geometryRotate2dBlock.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { Vector2 } from "core/Maths";
```
Comment (RaananW) [inline packages/dev/core/src/Meshes/Node/Blocks/geometryRotate2dBlock.ts:1]: Need to fix the import here (import from the file)

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14803/merge/testResults/webgpuplaywright/index.html

**Pull request merged**